src/highrl/envs/teacher_env.py

```diff
@@ -305,7 +349,7 @@ def step(self, action: List) -> Tuple:
             self.robot_id += 1
             # fmt: off
             print("initiating model ...")
-            model = PPO("MultiInputPolicy", self.robot_env, policy_kwargs=policy_kwargs, verbose=2, device=device)
+            model = PPO("MultiInputPolicy", self.robot_env, policy_kwargs=policy_kwargs, verbose=2, device=device,)
```

Comment:
Run `black` styling here to reduce the line length as follows
```python
model = PPO("MultiInputPolicy",
self.robot_env,
policy_kwargs=policy_kwargs,
verbose=2,
device=device,
)
```

Reply:
Done